packages/babel-standalone/src/preset-stage-2.ts

```diff
@@ -7,6 +7,8 @@ export default (_: any, opts: any = {}) => {
     useBuiltIns = false,
     decoratorsLegacy = false,
     decoratorsBeforeExport,
+    pipelineProposal = "hack",
```

Comment:
Unfortunately changing the default is a breaking change :sweat_smile: We cannot default to `hack` until Babel 8.

Reply:
Isn't that going to be really soon though?

Reply:
Soon as in "a few months, maybe end 2021"; this PR should be a quick merge (either next patch or next minor, I'm not sure if we should consider it as a bugfix or as a new feature for the stage 2 preset).

Reply:
Ok will revert it later tonight when I update PR.